Dockerfile

```diff
@@ -27,10 +27,10 @@ RUN cmake --build /BinaryCache/tensorflow-swift-apis --verbose
 
 # Clean out existing artifacts.
 # TODO: move into bash scripts...
-RUN rm /swift-tensorflow-toolchain/usr/lib/swift/linux/x86_64/TensorFlow.swiftinterface
-RUN rm /swift-tensorflow-toolchain/usr/lib/swift/linux/x86_64/TensorFlow.swiftdoc
-RUN rm /swift-tensorflow-toolchain/usr/lib/swift/linux/x86_64/TensorFlow.swiftmodule
-RUN rm /swift-tensorflow-toolchain/usr/lib/swift/linux/libswiftTensorFlow.so
+RUN rm -f /swift-tensorflow-toolchain/usr/lib/swift/linux/x86_64/TensorFlow.swiftinterface
```

Comment:
Note: I believe this logic was added when `TensorFlow` was built as part of the Swift standard library.
Its purpose is to delete `TensorFlow` toolchain artifacts so that the toolchain can be used to build [tensorflow/swift-apis](https://github.com/tensorflow/swift-api).

This may no longer be relevant when `TensorFlow` is no longer built as part of the standard library (ongoing effort), so we can simply remove these lines.